src/components/application_manager/src/rpc_passing_handler.cc

```diff
@@ -93,6 +93,74 @@ bool RPCPassingHandler::CanHandleFunctionID(int32_t function_id) {
   return false;
 }
 
+bool RPCPassingHandler::ExtractRPCParams(
+    const smart_objects::SmartObject& s_map,
+    const ApplicationSharedPtr app,
+    const std::string& function_id_str) {
+  LOG4CXX_AUTO_TRACE(logger_);
+  RPCParams params;
+
+  if (smart_objects::SmartType_Map == s_map.getType()) {
+    smart_objects::SmartMap::const_iterator iter = s_map.map_begin();
+    smart_objects::SmartMap::const_iterator iter_end = s_map.map_end();
```

Comment:
Probably don't need to make local variables for these iterators since they are only used in the for loop